knowrob_common/prolog/owl.pl

```diff
@@ -709,6 +788,79 @@
 owl_has_direct(S, P, O) :-
 	owl_use_rule(S, P, O).
 
+%% Simplest branch: find an explicitly stored rdf triple (S, P, O)
+owl_has_direct_internal(S, P, O) :-
+	rdf_has(S, P, O).
+
+%% If P is bound to an object property, see if any of its PropertyChain axioms is able to produce explicitly known triples.
+%% ASSUMPTION: no circular PropertyChain axioms (example, P defined as A o B and A defined as P o B)
+owl_has_direct_internal(S, P, O) :-
+        nonvar(P),
+        owl_individual_of(P, owl:'ObjectProperty'),
+        rdf_has(P, owl:propertyChainAxiom, RDFList),
+        kr_rdf_list(RDFList, PropChain),
+% We need this because the PropChain contains a list of either property or Description individuals (where a Description individual refers to an inverse property),
```

Comment:
For me it seems semantically wrong that these inverse property descriptions are not object properties themselves